addons/point_of_sale/tests/test_point_of_sale_flow.py

```diff
@@ -1650,11 +1650,13 @@ def test_invoicing_after_closing_session(self):
         reverser_customer_payment_entry = reverse_payment.line_ids.filtered(lambda l: l.account_id.account_type == 'asset_receivable')
         #check that both use the same account
         self.assertEqual(len(reverser_customer_payment_entry), 2)
+        self.assertEqual(len(original_customer_payment_entry), 2)
         self.assertTrue(order.account_move.line_ids.partner_id == self.partner1.commercial_partner_id)
         self.assertEqual(reverser_customer_payment_entry[0].balance, -4.0)
         self.assertEqual(reverser_customer_payment_entry[1].balance, -8.0)
         self.assertEqual(reverser_customer_payment_entry[0].amount_currency, -2.0)
         self.assertEqual(reverser_customer_payment_entry[1].amount_currency, -4.0)
+        self.assertEqual(self.partner1.total_due, 0.0)
```

Comment:
I remember that `total_due` is not present in `account` module. This test will fail if `account_followup` module is not installed. We can't put this check here.

Or, check if account_followup is installed, when you check the total due. Better, introduce a test module that checks total_due.